docs/reference/query-dsl/nested-query.asciidoc

```diff
@@ -49,22 +115,143 @@ GET /_search
+Adding the option `score_mode` allows you to set how inner children matching affects scoring of parent. It defaults to `avg`, but can be `sum`, `min`,
 `max` and `none`.
 
-There is also an `ignore_unmapped` option which, when set to `true` will
-ignore an unmapped `path` and will not match any documents for this query.
-This can be useful when querying multiple indexes which might have different
-mappings. When set to `false` (the default value) the query will throw an
-exception if the `path` is not mapped.
+The `ignore_unmapped` option set to `true` will ignore an unmapped `path` and will not match any documents for this query. This can be useful when querying multiple indexes which might have different mappings. Set to `false` (the default value) the query will throw an exception if the `path` is not mapped.
+
+==== Multi-level nesting
 
-Multi level nesting is automatically supported, and detected, resulting
-in an inner nested query to automatically match the relevant nesting
+Multi-level nesting is automatically supported and detected, resulting
+in an inner nested query that automatically matches the relevant nesting
 level (and not root) if it exists within another nested query.
+
+To set up a multi-level nested query, use this example. Notice that each level within the nest is separated by a dot (name.state.city).
+
+[source,js]
+--------------------------------------------------
+PUT my_index
+{
+    "mappings": {
+        "customers": {
+            "properties": {
+                "lastUpdated": {
+                    "type": "long"
+                },
+                "isDeleted": {
+                    "type": "boolean"
+                },
+                "person": {
+                    "type": "nested",
+                    "properties": {
+                        "car": {
+                            "type": "nested",
+                            "properties": {
+                                "model": {
+                                    "type": "string"
+                                },
+                                "make": {
+                                    "type": "string"
+                                }
+                            }
+                        },
+                        "last_name": {
+                            "type": "string"
+                        },
+                        "first_name": {
+                            "type": "string"
+                        }
+                    }
+                }
+            }
+        }
+    }
+}
+
+PUT my_index/_doc/1
+{
+    "person": {
+        "first_name": "Zach",
+        "last_name": "Foobar",
+        "car": [
+            {
+                "make": "Saturn",
+                "model": "SL"
+            },
+            {
+                "make": "Subaru",
+                "model": "Imprezza"
+            }
+        ]
+    },
+    "isDeleted": false,
+    "lastUpdated": 1433257051959
+}
+
+{
```

Comment:
This doesn't look quite right.

Reply:
Looks like `PUT customers/_doc/2` needs to be added before line 191

Reply:
Got it! That should help.

Reply:
> This doesn't look quite right.

Nik, this example came partially from a customer discussion. Are you saying that I need to remove the various   "lastUpdated": 1433257051959 lines, or that they need to be updated?